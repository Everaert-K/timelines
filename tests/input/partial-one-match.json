[
["sleep","cat"],
["awake","cat","dog"]
]