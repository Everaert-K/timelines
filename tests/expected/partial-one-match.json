[
["awake","cat","dog"],
["sleep","cat","dog"]
]
